ivy/functional/ivy/device.py

```diff
@@ -396,7 +400,7 @@ def clear_mem_on_dev(device: Union[ivy.Device, ivy.NativeDevice]) -> None:
         The device string to convert to native device handle.
 
     """
-    return current_backend(None).clear_mem_on_dev(device)
+    current_backend(None).clear_mem_on_dev(device)
```

Comment:
we should actually use `ivy.current_backend` for these calls, and remove the explicit import at the top of the file.